Form errors

qgerome opened the pull request.

pvanliefland reviewed (changes requested):
Looks good! But unless I missed something, the analysis name update is not covered.

Also a few console.log left...

qgerome commented:
> Looks good! But unless I missed something, the analysis name update is not covered.
> 
> Also a few console.log left...

Added the analysis name change and removed the console.log.
